classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -54,6 +54,8 @@
  *   required: Adds an asterisk and a .required class to the element's label
  */
 
+define('INSERT_TAG_VARIABLE_TYPE_PLAIN_TEXT', 'PLAIN_TEXT');
```

Comment:
Had to declare it in classes/core/PKPApplication.inc.php (I could import the classes/form/FormBuilderVocabulary.inc.php but preferred to declare it there)